WcaOnRails/app/models/competition_payment_integration.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# TODO: Add validation to ensure that multiple accounts of the same type don't get added to CompetitionPaymentIntegration
+class CompetitionPaymentIntegration < ApplicationRecord
+  belongs_to :connected_account, polymorphic: true
+
+  belongs_to :competition
+
+  AVAILABLE_INTEGRATIONS = {
+    'paypal' => 'ConnectedPaypalAccount',
+  }.freeze
+
+  scope :paypal, -> { where(connected_account_type: 'ConnectedPaypalAccount') }
```

Comment:
Nitpick: Use the hash?
```suggestion
  scope :paypal, -> { where(connected_account_type: AVAILABLE_INTEGRATIONS['paypal']) }
```

Reply:
If you adapt my suggestion from the other hash comment above, you need to access it here like `AVAILABLE_INTEGRATIONS[:paypal]`